openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx

```diff
@@ -287,9 +287,10 @@ const GlossaryPageV1 = () => {
   ) => {
     handleChildLoading(true);
     const hierarchy = getTermPosFromGlossaries(arrGlossary, dataFQN);
+    const isFQN = dataFQN;
```

Comment:
do we need this variable? can't we use dataFQN directly?

Reply:
@ShaileshParmar11  changes updated!.